CHANGELOG.md

```diff
@@ -61,6 +61,7 @@ Thanks to Raenore for the help on some of these features, and keyboardturner for
 - Tooltip texts across the addon were reworked to follow a more consistent format and add clarity when needed.
 - Chat links can now be closed with Escape, and profile chat link tooltips have received minor display improvements.
 - The Icon browser now displays icons in a larger size both on the grid and in tooltips.
+- Names and titles maximum length in tooltips has been reduced.
```

Comment:
Should update this to also mention race and class. Also, separate line pointing out that we now show tooltips for more truncated fields like names, titles, and traits?